tests/unit/test_management_service.py

```diff
@@ -191,6 +191,48 @@ def test_management_service_post_config_array_raises(management_service):
         })
 
 
+@mark.parametrize(
+    'request_body, expected_state',
+    [
+        ('{"enable": true}', True),
+        ('{"enable": false}', False),
+    ],
+)
+def test_management_service_set_recording(management_service, request_body, expected_state):
+    management_service.set_recording(
+        MITMRequest(
+            method='PUT',
+            url='http://mitm-manager.local/record',
+            body=request_body
+        )
+    )
+
+    result = management_service.is_recording
+
+    assert expected_state == result
+
+    for service in management_service.services:
+        assert expected_state == service.is_recording
```

Comment:
Add a test to make sure that the recording flag changes (set it to the opposite of what it is by default).

Reply:
Use the return value to check stuff